test/Microsoft.SqlTools.ServiceLayer.UnitTests/Microsoft.SqlTools.ServiceLayer.UnitTests.csproj

```diff
@@ -33,4 +33,8 @@
 	    <Content Remove=".\ShowPlan\TestExecution.xml" />
     	<EmbeddedResource Include=".\ShowPlan\TestExecutionPlan.xml" />
 	</ItemGroup>
+	<ItemGroup>
+	    <Content Remove=".\ShowPlan\TestExecutionPlanRecommendations.xml" />
+    	<EmbeddedResource Include=".\ShowPlan\TestExecutionPlanRecommendations.xml" />
```

Comment:
Align this correctly

Reply:
This seems to be aligned correctly on my end
![image](https://user-images.githubusercontent.com/6816294/151467575-588d17df-5eac-4d22-b204-989852fa0355.png)

I wonder why it looks weird over here

Reply:
Turn on whitespace visualization - it looks like there's a mix of tabs and spaces throwing things off. 

![image](https://user-images.githubusercontent.com/28519865/151468310-c0edbbbd-31b9-4c53-ba6f-0d5082b5cc80.png)